docs/user-guide/projects.md

```diff
@@ -54,7 +54,11 @@ argocd proj add-destination <PROJECT> <CLUSTER>,<NAMESPACE>
 argocd proj remove-destination <PROJECT> <CLUSTER>,<NAMESPACE>
 ```
 
-We can also do negations of destinations (i.e. install anywhere _apart from_).
+#### Deny destinations
+
+> v2.5
```

Comment:
should this be on the same line as `Deny destinations` like we've done elsewhere?

Reply:
Unfortunately I've seen a mix 😬